polaris-service/src/main/java/org/apache/polaris/service/TimedApplicationEventListener.java

```diff
@@ -66,16 +66,16 @@ private class TimedRequestEventListener implements RequestEventListener {
     @Override
     public void onEvent(RequestEvent event) {
       String realmId = CallContext.getCurrentContext().getRealmContext().getRealmIdentifier();
-      if (event.getType() == RequestEvent.Type.RESOURCE_METHOD_START) {
+      if (event.getType() == RequestEvent.Type.REQUEST_MATCHED) {
```

Comment:
FYI, this listener to `@TimedApi` (populated in mustache template) is not required with Quarkus.